### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Abelutzu** [CurimuChizu/CC-Themes#17]
So recently I downloaded the Re:Zero theme and the Discord Background was bugged ... How can I fix it ?
Dont mind the other photos ....
![e8848d90cd70b182c19a630d263adea8](https://user-images.githubusercontent.com/33329680/32344873-edfc35bc-c010-11e7-8837-20882c1a7513.png)

**Comment by UlithiumDragon** [comment]
Yeah this seems to have started happening again after the last silent Discord update. I tried looking at the code myself, but I don't know enough about CSS nor how Discord/BetterDiscord is set up to be able to fix it.

All I could confirm is that the issue is with the code itself, and not with the image host.

**Comment by CurimuChizu** [comment]
@UlithiumDragon @Abelutzu Yes. Unfortunately, I have given up on this project. Discord keeps changing their codings and I'm busy with school and other stuff. So I haven't touched this theme in a long time and I won't be updating it anytime soon. Sorry about that.

**Comment by FappyMcFap** [comment]
I made a few customized changes to keep everything purple. If you want it add this to custom css:

https://pastebin.com/J4ctpG8x

Feel free to change anything to your liking.
